usaspending_api/common/tests/test_drf_tracking.py

```diff
@@ -7,6 +7,7 @@
 import json
 
 
+@pytest.mark.skip(reason="Table logging is now disabled")
```

Comment:
Ditto the comment above--would prefer not to keep this stuff around if we're not using it.